### PR TITLE
ternary search code

### PR DESCRIPTION
**Pull request by Diksha4248**
### Checklist
🚨 Please review this repository's [contribution guidelines](https://github.com/ShubhKotnala/Data_Structures_And_Algorithms/blob/main/CONTRIBUTIONS.md).

- [ ] I've read and agree to the project's contribution guidelines.
- [ ] I'm requesting to **pull a topic/feature/bugfix branch**.
- [ ] I checked that my code additions will pass code linting checks and tests.
- [ ] I have added a comment block to support my contribution, explaning about the problem along with test cases.

### Description
What does this change do and why?

Thank you!

### CODE REVIEW

File: python/searching/TernarySearch.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+'''Ternary search is a decrease(by constant) and conquer algorithm that can be used to find an element in an array.
+It is same as the binary search. The only difference is that, it reduces the time complexity a bit more.
+Its time complexity is O(log n base 3) and that of binary search is O(log n base 2).'''
+
```
Comment (ShubhKotnala) [inline python/searching/TernarySearch.py:4]: Add a similar comment block in the other file as well

**Review by ShubhKotnala — CHANGES_REQUESTED** [review]
Please format the file, there are too many whitespaces

**Comment by Diksha4248** [comment]
> Please format the file, there are too many whitespaces

Done👍🏽

File: python/searching/TernarySearch.py (hunk 1 above)
Comment (Diksha4248) [reply python/searching/TernarySearch.py]: > Add a similar comment block in the other file as well

Okay , Done👍🏽

**Review by ShubhKotnala — APPROVED** [review]

**Pull request merged**